config/linux-x230.config

```diff
@@ -184,11 +183,10 @@ CONFIG_MFD_SYSCON=y
 CONFIG_DRM=y
 CONFIG_DRM_I915=y
 CONFIG_FB_VESA=y
-CONFIG_BACKLIGHT_LCD_SUPPORT=y
 # CONFIG_LCD_CLASS_DEVICE is not set
-CONFIG_BACKLIGHT_CLASS_DEVICE=y
```

Comment:
What is the reason for these changes?  I can’t see how they are related to USB devices.

Reply:
@DemiMarie : that is just the result of the savedefconfig. I could definitely revert the changes to only that module change.
I am not really ok with the standard of saving in savedefconfig for kernel configurations, this complicates migrating to newer kernel, since defaults change all the time.

Reply:
That’s up to you.